NotesApi/V1/Boundary/Response/NoteResponseObject.cs

```diff
@@ -1,16 +1,47 @@
 using NotesApi.V1.Domain;
 using System;
 
+
 namespace NotesApi.V1.Boundary.Response
 {
     public class NoteResponseObject
     {
         public Guid Id { get; set; }
+
         public string Description { get; set; }
+
         public TargetType TargetType { get; set; }
+
         public Guid TargetId { get; set; }
+
         public DateTime DateTime { get; set; }
```

Comment:
According to the Swagger this needs to be renamed to CreatedAt

Reply:
Should be changed all the way down to the db.